src/igvfd/searches/configs/alignment_file.py

```diff
@@ -108,6 +115,7 @@ def alignment_file():
                 'facet_fields': [
                     'file_set.file_set_type',
                     'assay_titles',
+                    'file_set.assay_term.term_name',
```

Comment:
You can literally delete `facet_groups` from every search config you touch. They don’t hurt anything, but they’re not used for anything either.